pubspec.yaml

```diff
@@ -5,7 +5,7 @@ repository: https://github.com/woltapp/wolt_modal_sheet
 
 environment:
   sdk: '>=3.0.6 <4.0.0'
-  flutter: 3.10.6
+  flutter: '>=3.13.0'
```

Comment:
```suggestion
  flutter: 3.13.0
```

Explicit version as we currently use

Reply:
3.13.0 has analyzer issues in the package?

Reply:
Please update our GitHub actions to use the same Flutter version.

Reply:
Explicit version also required by the PR check to use Flutter version from pubspec.

![Screenshot 2024-06-01 at 23 33 13](https://github.com/woltapp/wolt_modal_sheet/assets/48603081/e0e2ddf0-c08d-4fd8-8c2c-55c86bc7f302)

Reply:
@TahaTesser using this as 3.13.0 instead of '>=3.13.0' gives us warnings with pub dry run.

Reply:
@bartekpacia
You talked about `--dry-run` [here](https://github.com/subosito/flutter-action/discussions/215#discussioncomment-5382431),

if we add `>=3.13.0', it breaks our GitHub actions. Do you've any solutons?

Reply:
Hi @TahaTesser and @ulusoyca 👋 😁

Unfortunately, you cannot use version ranges (like `>= 3.13.0`) when also using `flutter-version-file`. [The stuff I talk in comment that you mentioned](https://github.com/subosito/flutter-action/discussions/215#discussioncomment-5382431) is actually not implemented in https://github.com/subosito/flutter-action/pull/290.

Sorry about that – and PRs are welcome :)

Reply:
@bartekpacia  Appreciate it!

@ulusoyca 
Could you please update GitHub actions with `3.13.0` and remove automatic version tag to fix our checks?